packages/block-library/src/comment-template/test/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Internal dependencies
+ */
+import { convertToTree } from '../util';
+
+describe( 'Convert to tree', () => {
+	it( 'Empty comments', () => {
+		const comments = convertToTree( [] );
+
+		expect( comments ).toEqual( [] );
+	} );
+
+	it( 'Handle comments', () => {
+		const comments = convertToTree( [
+			{ id: 1, parent: 0 },
+			{ id: 2, parent: 0 },
+			{ id: 3, parent: 2 },
+			{ id: 4, parent: 2 },
+			{ id: 5, parent: 4 },
+			{ id: 6, parent: 1 },
+		] );
+
+		expect( comments ).toEqual( [
+			{
+				id: 1,
+				commentId: 1,
```

Comment:
Do we need both `id` and `commentId`, at least in the test they are always the same.

Reply:
Oh, nice catch, that's my omission. I was planning to pass the `id` to each individual comment so that they can be linked to. E.g. a comment with `id` of `8` would be linked to like: http://mysite.blog/?p=11#comment-8. But I can do the same with just the `commentId`, of course 😃

Reply:
Done in 802e37973f15681ea146eb9380f973b39c0d11b3